semeio/jobs/scripts/design2params.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+#
+import argparse
+import logging
+import os
+import sys
+
+from semeio import valid_file
+from semeio.jobs.design2params import design2params
+
+description = """
+Reads a design matrix in XLSX-format and:
+ * Converts specified worksheet to txt-format and puts designmatrix.txt in RUNPATH
+ * Fetches values belonging to given ERT realization and *appends*
+   key-value pairs to parameters.txt
+ * Creates designparameters.txt which only contains variables taken from the design matrix, this
+   can be given to CUSTOM_KW
+Requires a matrix with column header as strings in topmost row in Excel.
+Row 2 in Excel must then correspond with the data you want for 'realization 0'
+Column 1 in Excel should contain the realization number, and will be ignored by this script.
+You must run this script as a FORWARD_MODEL from your ERT config, before
+you run DESIGN_KW.
+
+Usage:
+ $ design2params.py <realization-id> <xls-filename> <worksheet-name> [<defaultssheetname>]
+defaultssheetname is an optional argument, you may choose not to provide it (just give an empty string)
+"""
+
+
+def parse_args(args):
+    parser = argparse.ArgumentParser(description=description)
+
+    parser.add_argument(
+        "realization", type=int, help="Which realization",
```

Comment:
When I use a different realization than `1` the job fails, example of failing test:
```
@pytest.mark.usefixtures("input_data")
@pytest.mark.parametrize("realization_id", [(1), (2), (3)])
@pytest.mark.parametrize(
    "test_file, expected_file",
    [
        (design2params._PARAMETERS_TXT, "refparameters_w_default.txt"),
        (design2params._DESIGN_MATRIX_TXT, "refdesignmatrix.txt"),
        (design2params._DESIGN_PARAMETERS_TXT, "refdesignparameters_w_default.txt"),
    ],
)
def test_run_with_default(test_file, expected_file, realization_id):
    design2params.run(
        realization_id,
        "design_matrix_tampered_defaults.xlsx",
        "DesignSheet01",
        "DefaultValues",
        log_level=logging.DEBUG,
    )

    with open(design2params._TARGET_FILE_TXT, "r") as status_file:
        status = status_file.read()
        assert status == "OK\n"
```